modular_citadel/code/modules/arousal/organs/genitals.dm

```diff
@@ -343,3 +368,75 @@
 
 	for(var/L in relevant_layers)
 		H.apply_overlay(L)
+
+/* Behold the maddness I went through. This is here until the genetial problem is confirmed fixed.
```

Comment:
Your PR is goddamn full of commented stuff.
I do hope you ut out what's useless when you finalize this.
And I also hope you find some better naming pattern for a couple of procs, path names etcetera instead of slapping in Fermi all around.

Reply:
I comment a lot to track changes I'm making, I'm in the process of scrubbing it.

Reply:
It's fair, I just say this to encourage you to keep this PR "clean".
This is still a WIP, so I won't blame you for having lot and lot of commented stuff right now, 
but this may be an issue when it'll be ready for review.

Reply:
Thanks for pointing it out though, I'll leave in the comments for maintainers to read, describing how the system works, unless you think it's best left out?

Reply:
Up to you, personally commented code is better removed if it without use and good purpose.